src/components/note/__internal__/note.style.js

```diff
@@ -45,42 +45,37 @@ const StyledTitle = styled.header`
 `;
 
 const StyledFooterContent = styled.div`
-  position: relative;
   line-height: 21px;
-
-  &:first-of-type {
-    font-weight: bold;
-    font-size: 14px;
-    top: 16px;
-    height: 21px;
-  }
-
   ${({ theme }) => `
+    &:first-of-type {
+      font-weight: bold;
+      font-size: 14px;
+      margin-top: ${2 * theme.spacing}px;
+    }
+
     &:nth-of-type(2) {
       font-weight: bold;
       font-size: 12px;
       color: ${theme.note.timeStamp};
-      left: 16px;
-      top: 17px;
-      height: 21px;
+      margin-left: ${2 * theme.spacing}px;
+      margin-top: ${1 + (2 * theme.spacing)}px;
     }
 
     &:last-of-type:not(:nth-of-type(2)) {
       font-weight: bold;
       font-size: 12px;
       color: ${theme.note.timeStamp};
       cursor: pointer;
-      top: 17px;
-      height: 21px;
-      left: 40px;
+      margin-top: ${1 + (2 * theme.spacing)}px;
```

Comment:
Same question as above, I think we should either use a multiple of `theme.spacing` or just use a px value